modules/nw-ovn-ipsec-traffic.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-ovn-ipsec-traffic_{context}"]
+= Types of network traffic flows encrypted by IPsec
+
+With IPsec enabled, only the following network traffic flows between pods are encrypted:
+
```

Comment:
We might need to re-visit this section. As pr testing following is encrypted or not:
Left side is traffic originator and right side is receiver as follow
```
Hostnw-pod -> pod is encrypted
pod -> pod is encrypted
pod-> hotnw-pod in NOT
Hostnw-pod ->Hostnw-pod is NOT
```

Reply:
Yeah, I think you are right. I even read it and through it was ok. Maybe a table like what @anuragthehatter has would be clearer.

Reply:
@markdgray, I'm going to put in a request for a diagram and include that, with the enablement slides as a template for that. (There are certain requirements for public facing assets, so they're created from scratch.)

Reply:
@jboxman Another thing that we should probably include is that this feature is for single stack IPv4 clusters only

Reply:
@markdgray, will that be true when dual stack is available?

Reply:
@markdgray @anuragthehatter 

And to clarify my understanding, given:

- _Pod A_, A pod on the host network
- _Pod Z_, A pod on the cluster network

If Pod A send an HTTP request to to Pod Z, the HTTP request would traverse the IPsec tunnel, but the response would not traverse the tunnel? (So the traffic is encrypted unidirectionally? Which is to say also that the IP traffic follows a different route, inbound and outbound on the network infrastructure?)

Reply:
> @markdgray, will that be true when dual stack is available?

Yes, we will explicitly enable this

Reply:
> And to clarify my understanding, given:
> 
>     * _Pod A_, A pod on the host network
> 
>     * _Pod Z_, A pod on the cluster network
> 
> 
> If Pod A send an HTTP request to to Pod Z, the HTTP request would traverse the IPsec tunnel, but the response would not traverse the tunnel? (So the traffic is encrypted unidirectionally? Which is to say also that the IP traffic follows a different route, inbound and outbound on the network infrastructure?)

No, what is important is who initiates the connection. Pod A->Pod Z will get SNAT'ed on the way out of the node so an return communications would go over the pod network and get encrypted.

Reply:
@anuragthehatter, does it look okay now?

Thanks!

Reply:
@jboxman Could you guide me the right path on netlify to read that? I tried to find under networking but couldn't

Reply:
@anuragthehatter should be: https://deploy-preview-27911--osdocs.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/about-ipsec-ovn.html

Reply:
Thanks Jason. That looks good to me.